src/go/k8s/tests/e2e/pandaproxy-produce-consume-tls-client-external-ca/02-produce-message.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: produce-message
+spec:
+  template:
+    spec:
+      volumes:
+        - name: client-cert
+          secret:
+            secretName: client-cert
+      containers:
+        - name: rpk
+          image: localhost/redpanda:dev
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
+            - mountPath: "/tmp/client"
+              name: client-cert
+              readOnly: true
+          command:
+            - /bin/bash
+            - -c
+            - -ex
+          args:
+            - |
+              curl -skv -X POST "https://cluster-proxy-0.cluster-proxy.$POD_NAMESPACE.svc.cluster.local:8082/topics/my_topic" \
+              -H "Content-Type: application/vnd.kafka.json.v2+json" \
+              --cert /tmp/client/client.crt --key /tmp/client/client.key --sslv2 --http2 \
+              -d '{"records":[{"value":"Vectorized"},{"value":"Pandaproxy"},{"value":"JSON Demo"}]}'
```

Comment:
Did you know you don't need to have backslashes if you do it this way?

```suggestion
          args:
            - >
              curl -skv -X POST "https://cluster-proxy-0.cluster-proxy.$POD_NAMESPACE.svc.cluster.local:8082/topics/my_topic"
              -H "Content-Type: application/vnd.kafka.json.v2+json"
              --cert /tmp/client/client.crt --key /tmp/client/client.key --sslv2 --http2
              -d '{"records":[{"value":"Vectorized"},{"value":"Pandaproxy"},{"value":"JSON Demo"}]}'
```

Reply:
Cool. Thank you!